src/utils/util.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+
+from PyDictionary import PyDictionary as PyDict
+
+
+class SynonymsUtil(object):
+
+    def __init__(self):
+        self.__py_dict = PyDict()
+
+    def find_synonyms(self, word: str, synonyms_count: int = 5):
```

Comment:
It is not obvious that this function also returns word, it is confusing when reading code of `__parse_news_by_keywords_and_synonyms`. It could return empty list or list of synonyms and be merged in `__parse_news_by_keywords_and_synonyms` like 
```
if cls.__news_content_contains_any_word(news_content, keyword + keywords_with_synonyms[keyword]):
                    parsed_news_with_keywords[news].append(keyword).
```
Then the name of keywords_with_synonyms could be changed to keywords_to_synonyms as you said.

Reply:
resolved